config/environments/production.rb

```diff
@@ -85,7 +85,7 @@
   #config.lograge.enabled = true
 
   # Use the Sidekiq adapter for Active Job if configured in settings
-  if Settings.background_jobs == true
+  if Settings.background_jobs.enabled == true
```

Comment:
FWIW: `Settings.background_jobs` isn't in the default `settings.yml`.  so, for safety's sake, it seems like the above should be `if Settings.background_jobs && Settings.background_jobs.enabled == true`, or the default `settings.yml` should have a `background_jobs` section with `enabled` set to `false`.

apologies if you're already on that, i know this was an in-the-middle-of-testing fix, but figured it was worth pointing out just in case...